README.md

```diff
@@ -26,6 +26,42 @@ The monorepo is split into separate packages automatically:
 **Please do not use `contao/contao` in production** but use the split packages
 instead.
 
+## Test setup
+
+To create a Pull Request to this monorepository and test your changes within
+a running Contao 4 application it is easiest to use the [Contao Managed Edition][3].
+Start by installing it in your current directory running
+
+```
+$ composer create-project contao/managed-edition .
```

Comment:
Since this will also install the dependencies, it would be better to do a

```
wget https://raw.githubusercontent.com/contao/managed-edition/master/composer.json
```

wouldn't it?

Reply:
I like to work with one tool only but we might wanna pass the `--no-install` flag?

Reply:
Yes, if that only downloads the `composer.json` file. 👍